index.js

```diff
@@ -0,0 +1,17 @@
+var express = require('express')
+var app = express()
+var bodyParser = require('body-parser')
+var parseRequests = bodyParser.json()
+var pets = require('./pets').pets
```

Comment:
Once we change `pets.js` to `module.exports = [...]`, this will now be `var pets = require('./pets')`